helm/openwhisk/values.yaml

```diff
@@ -205,7 +205,7 @@ invoker:
         enabled: false
         port: 3233
   containerNetwork: "bridge"
-  kubeDNS: nil
+  kubeDNS: ""
```

Comment:
Here I don't see `kubeDNS` is under `containerFactory`. What I see here is `kubeDNS` is in the same level as `containerFactory`. I don't understand why you use `.Values.invoker.containerFactory.kubeDNS` in `invoker-pod.yaml`.